test-requirements.txt

```diff
@@ -0,0 +1 @@
+odoo-addon-stock_analytic@ git+https://github.com/oca/account-analytic@refs/pull/448/head#subdirectory=setup/stock_analytic
```

Comment:
You can remove this commit. That PR is already merged. That should leave the CI OK.